addons/project/static/src/views/project_task_kanban/project_task_kanban_header.js

```diff
@@ -46,10 +46,15 @@ export class ProjectTaskKanbanHeader extends KanbanHeader {
         if (this.group.groupByField.name === 'stage_id') {
             const action = await this.group.model.orm.call(
                 this.group.resModel,
-                'unlink_wizard',
+                'remove_task_stage',
                 [this.group.resId],
                 { context: this.group.context },
             );
+            if (action == true) {
```

Comment:
```suggestion
            if (!action) {
```
This would be the change based on the previous comment